autopilot/betweenness_centrality_test.go

```diff
@@ -88,7 +89,16 @@ func TestBetweennessCentralityWithNonEmptyGraph(t *testing.T) {
 	workers := []int{-1, 1, 3, 9, 100}
 
 	expectedCentrality := []float64{
-		3.0, 0.0, 3.0, 15.0, 6.0, 6.0, 7.0, 0.0, 0.0,
+		0.2, 0.0, 0.2, 1.0, 0.4, 0.4, 0.466, 0.0, 0.0,
+	}
+
+	// almostEq returns true of a and b are close enough.
+	almostEq := func(a, b float64) bool {
```

Comment:
is this just needed to handle FPA?

Reply:
Yeah, it's a bit impractical to implement so removed for now and used a division instead.